Makefile

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DESTBINDIR = /usr/local/bin
+
+KATA_RUNTIME_NAME = kata-runtime
+KATA_RUNTIME_PATH = $(DESTBINDIR)/$(KATA_RUNTIME_NAME)
+
+ifeq (,$(KATA_RUNTIME))
+    # no argument specified
+    ifeq (,$(MAKECMDGOALS))
+        fail = true
+    endif
+
+    # remove goals that don't require the variable to be set
+    remaining=$(filter-out help,$(MAKECMDGOALS))
+
+    ifneq (,$(remaining))
+        fail = true
+    endif
+
+    ifeq ($(fail),true)
+        $(error "ERROR: KATA_RUNTIME not set - run 'make help'")
+    endif
+endif
+
+ifeq (cc,$(KATA_RUNTIME))
+	RUNTIME_DIR = cc-runtime
+	RUNTIME_NAME = kata-runtime-cc
+	TARGET = $(RUNTIME_NAME)
+	DESTTARGET = $(DESTBINDIR)/$(TARGET)
+endif
+
+ifeq (runv,$(KATA_RUNTIME))
+	RUNTIME_DIR = runv
+	RUNTIME_NAME = runv
```

Comment:
should this be kata-runtime-runv?

Reply:
Nevermind -- I see that our runtime name is updated just to avoid confusion and we ultimately make the symlink regardless.

Thanks James -- looks fine to me....

Reply:
Probably, but `runv`'s build doesn't allow the target name to be overridden unfortunately.